core/optaplanner-constraint-streams/src/test/java/org/optaplanner/constraint/streams/uni/UniConstraintStreamTest.java

```diff
@@ -2142,44 +2142,6 @@ public void zeroConstraintWeightDisabled() {
         assertThat(oneWeightMonitorCount.get()).isEqualTo(1);
     }
 
-    @TestTemplate
-    public void nodeSharing() {
```

Comment:
Why not keep this test? It's a high level integration test.
NodeSharingTest is a low level unit test. Might be good to keep both?

Reply:
OK. It will work.